Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/Search.hs

```diff
@@ -52,26 +53,51 @@ buildSearchReq subscriber req = do
   let disabilityTag = buildDisabilityTag =<< intent.fulfillment.customer
   let messageId = context.message_id
   transactionId <- context.transaction_id & fromMaybeM (InvalidRequest "Missing transaction_id")
-  pure
-    DSearch.DSearchReq
-      { messageId = messageId,
-        transactionId = transactionId,
-        bapId = subscriber.subscriber_id,
-        bapUri = subscriber.subscriber_url,
-        bapCity = context.city,
-        bapCountry = context.country,
-        pickupLocation = LatLong {lat = pickup.location.gps.lat, lon = pickup.location.gps.lon},
-        pickupTime = now,
-        dropLocation = LatLong {lat = dropOff.location.gps.lat, lon = dropOff.location.gps.lon},
-        pickupAddress = pickup.location.address,
-        dropAddrress = dropOff.location.address,
-        routeDistance = distance,
-        routeDuration = duration,
-        device = Nothing,
-        routePoints = buildRoutePoints =<< intent.fulfillment.tags, --------TODO------Take proper input---------
-        customerLanguage = customerLanguage,
-        disabilityTag = disabilityTag
-      }
+  pure $
+    case dropOff of
```

Comment:
this case match shouldn't happen on base of toLocation. means toLocation shouldn't be deciding factor for onDemand or Rental.

Reply:
Agree, should be some tag or something like this

Reply:
Done